### PR TITLE
Cynerio add workbooks

### PR DESCRIPTION
**Pull request by meravcy**
Required items, please complete
   
   Change(s):
   - Add workbook and analytic rules

   Reason for Change(s):
   - Added content to solution

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - No

### CODE REVIEW

**Comment by v-rbajaj** [comment]
Hi @meravcy, please repackage the solution using this tool https://github.com/Azure/Azure-Sentinel/tree/0cc258c900726e89aaf8b39984be98d9a7a3cc1a/Tools/Create-Azure-Sentinel-Solution/V3
Please go through the readme.md file and follow the instructions mentioned there.

And only add version 3.0.0 zip in this PR.

**Comment by meravcy** [comment]
Hi @v-rbajaj , created v3 package and added release notes

**Comment by meravcy** [comment]
Hi @v-rbajaj , can you please help with the arm-ttk validation test that fails? I couldn't understand what's wrong with the workbook, it's a very simple one, exported json and looks like other workbooks exist in the repository

**Comment by v-rbajaj** [comment]
Hi @meravcy, to fix the failing ARM TTK validation we would need to add the details of the new workbook in WorkbooksMetadata --> https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json) and then repackage the solution (we don't have to increment the version of the solution)

**Comment by v-rbajaj** [comment]
Hi @meravcy, 
Below are few things which I have noted about the analytical rules (applies to all the rules)
They are few properties like "tags" etc which are missing and also sequencing isn't proper, please check this analytical rule for the same - https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/DNS%20Essentials/Analytic%20Rules/ExcessiveNXDOMAINDNSQueriesAnomalyBased.yaml

File: Solutions/Cynerio/Analytic Rules/IoTDefaultPasswords.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+id: 84e0ea1f-766d-4775-836a-c0c9cca05085
+name: Cynerio - IoT - Default password
+description: IoT Default Passwords
```
Comment (v-rbajaj) [inline Solutions/Cynerio/Analytic Rules/IoTDefaultPasswords.yaml:3]: Can you please make the description more descriptive?

File: Solutions/Cynerio/Analytic Rules/IoTExploitationAttempts.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+id: 3d853a88-92d2-4aec-a680-2bf7bb560c56
+name: Cynerio - Exploitation Attempt of IoT device
+description:  IoT Exploitation Attempt
```
Comment (v-rbajaj) [inline Solutions/Cynerio/Analytic Rules/IoTExploitationAttempts.yaml:3]: Can you please make the description more descriptive?

File: Solutions/Cynerio/Analytic Rules/IoTWeakPasswords.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+id: 65db1346-6435-4079-bbf4-9a7113c98054
+name: Cynerio - IoT - Weak password
+description:  IoT Weak Password
```
Comment (v-rbajaj) [inline Solutions/Cynerio/Analytic Rules/IoTWeakPasswords.yaml:3]: Can you please make the description more descriptive?

File: Solutions/Cynerio/Analytic Rules/IoTExploitationAttempts.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+requiredDataConnectors:
+  - connectorId: CynerioSecurityEvents
+    dataTypes:
+      - CynerioEvent_CL
+queryFrequency: 5h
+queryPeriod: 5h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - "LateralMovement"
```
Comment (v-rbajaj) [inline Solutions/Cynerio/Analytic Rules/IoTExploitationAttempts.yaml:14]: Please remove the double quotes from "LateralMovement"

**Comment by meravcy** [comment]
Hey @v-rbajaj , resolved all discussions

**Review by v-rbajaj — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**